cmd/agent/install_script.sh

```diff
@@ -220,7 +221,26 @@ elif [ "$OS" = "Debian" ]; then
     fi
     printf "\033[34m\n* Installing APT package sources for Datadog\n\033[0m\n"
     $sudo_cmd sh -c "echo 'deb https://${apt_url}/ ${apt_repo_version}' > /etc/apt/sources.list.d/datadog.list"
-    $sudo_cmd apt-key adv --recv-keys --keyserver "${keyserver}" A2923DFF56EDA6E76E55E492D3A80E30382E94DE
+    retries=0
+    while [ "$retries" -le 4 ]; do
```

Comment:
You could use:

```
for retries in {0..4}; do
...
done
```

That saves you the variable initialization and incrementation.